hilti/toolchain/src/config.cc.in

```diff
@@ -106,13 +106,14 @@ void Configuration::init(bool use_build_directory) {
     //
     // (3) it helps the optimizer to know that symbols won't be accessed
     // externally.
-    runtime_cxx_flags_debug = flatten({"-fPIC", "-std=c++17", "-g", "-fvisibility=hidden",
+    runtime_cxx_flags_debug = flatten({"-fPIC", "-std=c++17", "-g", "-fvisibility=hidden", "-pthread",
```

Comment:
There's the previous  commit: https://github.com/zeek/spicy/commit/109b58c044083a313729667380bc6adfeafd3ec6

I'm wondering if this now means we can switch back to giving `-pthread` to the linker (instead of `-lpthread`. Before that didn't work because the precompiled headers wouldn't get that flag.

Reply:
This part is actually not needed anymore as I decided to not link against their static library and instead include it in our cpp file (otherwise they would always insist on installing their headers). Should I just get rid of above pthread change or did I accidentally fix some other issue?

Reply:
Dropped the patch.